test/e2e/specs/blocks/blocks__smoke-test-across-gb-edge-versions.ts

```diff
@@ -15,16 +15,26 @@
  * To avoid any confusion, the tests here will only run if the GUTENBERG_EDGE env
  * var is set.
  */
-import { GutenbergEditorPage, TestAccount, envVariables, skipItIf } from '@automattic/calypso-e2e';
+import {
+	GutenbergEditorPage,
+	TestAccount,
+	envVariables,
+	skipItIf,
+	buildDefaultTestTargetsTable,
+} from '@automattic/calypso-e2e';
 import { Page, Browser } from 'playwright';
 
 declare const browser: Browser;
 
-describe.each`
-	siteType      | accountName                        | testPostId
-	${ 'Simple' } | ${ 'gutenbergUpgradeEdgeUser' }    | ${ 42805 }
-	${ 'Atomic' } | ${ 'gutenbergAtomicSiteEdgeUser' } | ${ 32 }
-`(
+type AdditionalRecord = { testPostId: number };
+
+describe.each(
+	buildDefaultTestTargetsTable< AdditionalRecord >( {
+		edge: true,
+		additionalRecords: [ { testPostId: 42805 }, { testPostId: 32 } ],
+		activationCriteria: { Atomic: false, Simple: true },
```

Comment:
This is just for illustration purposes, I'll remove the `activationCriteria` soon as this test doesn't need this.

Reply:
The idea of this param is to allow some tests to activate a target based on external factors (i.e an env var). The gist is that not all tests will need to run on AT all the time.